### PR TITLE
change default write buffer size of lockcf to 32MB

### PR DESCRIPTION
**Pull request by Little-Wallace**
###  What have you changed?

I set `write_buffer_size` of lockcf 32MB, whose previous default value is 128MB.
When TiKV write keys frequently into a TiKV node, there will be many tombstones in lockcf. If user want scan some keys  which are just inserted into TiKV, it would cost much CPU. Such as:
`SELECT * from tableA order by create_time desc limit 100`.
So we decrease size of memtable in lockcf to optimize to reduce cost of range query which include many tombstones.

###  What is the type of the changes?

- Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?

- No code

###  Does this PR affect `tidb-ansible`?

Yes.

### CODE REVIEW

**Comment by 5kbpers** [comment]
Will it introduce potential write stall problem?

**Comment by zhangjinpeng1987** [comment]
Please also change the value in `etc/config-template.toml`.

File: etc/config-template.toml [hunk 1]
```diff
@@ -665,7 +665,7 @@
 # write-buffer-size = "128MB"
```
Comment (zhangjinpeng1987) [inline etc/config-template.toml]: I think you should change write-buffer-size, not max-bytes-for-level-base.

Comment (Little-Wallace) [reply etc/config-template.toml]: Sorry, I made a mistake.

File: etc/config-template.toml [hunk 2]
```diff
@@ -96,7 +96,7 @@
 # grpc-memory-pool-quota = "32G"
 
 ## The number of connections with each TiKV server to send Raft messages.
-# grpc-raft-conn-num = 10
+# grpc-raft-conn-num = 1
```
Comment (shenli) [inline etc/config-template.toml]: why do you change this to one?

Comment (5kbpers) [reply etc/config-template.toml]: @shenli I saw that this default value is actually 1 in our code

**Pull request merged**